riak_test/yz_rt.erl

```diff
@@ -291,6 +293,37 @@ load_data(Cluster, Bucket, YZBenchDir, NumKeys, Operations) ->
 load_data(Cluster, Bucket, YZBenchDir, NumKeys) ->
     load_data(Cluster, Bucket, YZBenchDir, NumKeys, [{load_fruit, 1}]).
 
+
+-spec write_data(cluster(), bucket(), non_neg_integer(), non_neg_integer()) ->
+    [ok | {error, term()}].
+write_data(Cluster, Bucket, Start, Count) ->
+    ValueGenerator = fun(Key) -> {Key, "text/plain"} end,
+    write_data(Cluster, Bucket, Start, Count, ValueGenerator).
+
+-spec write_data(cluster(), bucket(), non_neg_integer(), non_neg_integer(), fun((non_neg_integer()) -> ok | term()))  ->
```

Comment:
rt already has a version of generating data, and this one has specific support for mime types.  We could, but I don't think it's necessary.